CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.6.1 (2019-09-24)
+
+- Updated moment imports to avoid build failures. [#85](https://github.com/blackbaud/skyux-datetime/pull/85)
```

Comment:
To avoid or prevent? If the import is stopping the build failures from happening altogether, I think we should use prevent.

Reply:
I would spin it to be more positive: "Fixed `moment` imports to support backwards compatibility with SKY UX 2 single-page applications." Something like that.

Reply:
I like Steve's suggestion!

Reply:
Done!